docs/DeveloperGuide.md

```diff
@@ -36,3 +44,64 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Parser Component
+
+Here is a general guide of how Parse Class looks like in sequence diagram.
+
+![Parse.png](diagram%2FParse.png)
+
+Here is a class diagram for the method changeTaskTiming method in Parser Class.
+
+![changeTaskTimingClassDiagram.png](diagram%2FchangeTaskTimingClassDiagram.png)
+
+The following sequence diagram of changeTaskTiming shows the interactions with other objects.
+
+![changeTaskTiming_Sequence_Diagram.png](diagram%2FchangeTaskTiming_Sequence_Diagram.png)
```

Comment:
Good sequence diagram with explanations below. Consider the following improvements:
- Adding an activation bar to `UserList`, `Timetable`
- Adding the relevant return arrow (if necessary)
- Since there are explanations below, it would be nice to "tag" the index in the description above the arrow. eg. `1. changeTaskTiming(command, userList)` , which would match to bullet point 1 in the explanation section below.

Reply:
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/55198236/befd986b-57c8-4fb3-aed4-b2c87e791218)

The return arrow seems to have been generated by the wrong class. Can consider checking if there is a mistake there.